<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>1374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArkShopHelper</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 15pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_firstpage" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>350</width>
      <height>600</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	
	font: 20pt &quot;楷体&quot;;
}</string>
     </property>
     <property name="text">
      <string>首页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_shoppage" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>350</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	
	font: 20pt &quot;楷体&quot;;
}</string>
     </property>
     <property name="text">
      <string>商店</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>351</width>
       <height>571</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shopState">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_kitpage" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>630</y>
      <width>350</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	
	font: 20pt &quot;楷体&quot;;
}</string>
     </property>
     <property name="text">
      <string>礼包</string>
     </property>
    </widget>
    <widget class="QListView" name="kitViewList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>351</width>
       <height>571</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="kitState">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_mypage" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>350</width>
      <height>600</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	
	font: 20pt &quot;楷体&quot;;
}</string>
     </property>
     <property name="text">
      <string>我的</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="steamidLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>未登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>金币：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>VIP等级：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pointsLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="vipLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateDataButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>更新个人信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bdqqButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>绑定QQ号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setPassButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>设置登录密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>F2
显示/隐藏</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_serverlink" native="true">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>350</width>
      <height>600</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	
	font: 20pt &quot;楷体&quot;;
}</string>
     </property>
     <property name="text">
      <string>直连</string>
     </property>
    </widget>
    <widget class="QLabel" name="linkState">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>pushButton_2</zorder>
   <zorder>widget_kitpage</zorder>
   <zorder>widget_mypage</zorder>
   <zorder>widget_firstpage</zorder>
   <zorder>widget_shoppage</zorder>
   <zorder>label_9</zorder>
   <zorder>widget_serverlink</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
